apps/safe-claiming-app/src/widgets/ClaimingWidget.tsx

```diff
@@ -113,14 +128,43 @@ const ClaimingWidget = () => {
   const ctaWidget = (
     <>
       <div>
-        <Title>Become the part of Safe's future!</Title>
+        <Title>Become part of Safe's future</Title>
         <br />
         <Subtitle>
-          You will be able to buy $SAFE once the token transferability is
-          enabled.
+          Help us unlocking ownership for everyone by joining the discussions in
+          the{" "}
+          <StyledExternalLink
+            href={FORUM_URL}
+            rel="noreferrer noopener"
+            target="_blank"
+            variant="subtitle1"
+            textAlign="center"
+            fontSize="small"
+          >
+            SafeDAO Forum
+            <OpenInNewRounded
+              sx={{ width: "0.75em", height: "0.75em" }}
+              fontSize="small"
+            />
+          </StyledExternalLink>{" "}
```

Comment:
I'd suggest creating a component for these as they are the same as each other. They can stay in this file though.